src/client/src/ui/spotTile/components/PriceMovement.tsx

```diff
@@ -10,28 +10,27 @@ interface Props {
 
 const MovementIcon = styled('i')<{ show: boolean; color: string }>`
   text-align: center;
-  color: ${({ theme: { palette }, color }) => palette[color].normal}}};
+  color: ${({ theme, color }) => theme[color].base}}};
   visibility: ${({ show }) => (show ? 'visible' : 'hidden')};
 `
 
 const MovementValue = styled('div')`
   font-size: 11px;
-  color: ${({ theme: { text } }) => text.textMeta};
+  opacity: 0.59;
```

Comment:
Same as above — test the corresponding palette value.

We can also add a function to get the nearest value from our palette given a base color and an opacity … if you like.